src/structures/Message.js

```diff
@@ -127,25 +127,28 @@ class Message extends Base {
      * lost if re-fetched</warn>
      * @type {?string}
      */
-    this.nonce = 'nonce' in data ? data.nonce : null;
+    this.nonce = partial ? this.nonce : 'nonce' in data ? data.nonce : null;
```

Comment:
`if (!partial)`?

Reply:
Sure, will use if statements to make it clear